### PR TITLE
updated datastream xml files for the test content image

### PR DESCRIPTION
**Pull request by Vincent056**
There was a regression in the test content image `quay.io/compliance-operator/test-broken-content:kublet_default` on OCP 4.12 cluster, which caused the e2e test to fail. Since we have fix the test image, here we updated the datastream xml files for the test content image.

### CODE REVIEW

**Comment by rhmdnd** [comment]
This should fix https://github.com/ComplianceAsCode/compliance-operator/issues/164 right?

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,7 +24,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Added a utility script to make it easier for maintainers to propose releases,
   regardless of the git remote configuration. See the previously closed
   [issue](https://github.com/ComplianceAsCode/compliance-operator/issues/8) for
-  more details.
+
+- There was a regression in `quay.io/compliance-operator/test-broken-content:kublet_default`
```
Comment (rhmdnd) [inline CHANGELOG.md:28]: Should this be kubelet_default?

Comment (Vincent056) [reply CHANGELOG.md:28]: that's correct, let me fix that

**Review by JAORMX — CHANGES_REQUESTED** [review]
Trivy seems unhappy with a dependency. You need to run:
```
go get -u golang.org/x/text@0.3.8
go mod tidy && go mod verify && go mod vendor
```
To address this.

**Comment by jhrozek** [comment]
> Trivy seems unhappy with a dependency. You need to run:
> 
> ```
> go get -u golang.org/x/text@0.3.8
> go mod tidy && go mod verify && go mod vendor
> ```
> 
> To address this.

Thanks, I opened  PR #166 to address this. Would you mind acking this so that @Vincent056 can rebase his PR and we can get the release out of the door?

**Comment by rhmdnd** [comment]
> Thanks, I opened PR #166 to address this. Would you mind acking this so that @Vincent056 can rebase his PR and we can get the release out of the door?

I approved #166. Should be good to rebase this soon.

**Comment by Vincent056** [comment]
> > Thanks, I opened PR #166 to address this. Would you mind acking this so that @Vincent056 can rebase his PR and we can get the release out of the door?
> 
> I approved #166. Should be good to rebase this soon.

Thanks for updating this, just rebased the PR here

**Review by rhmdnd — APPROVED** [review]
/lgtm

**Pull request merged**